lib/team.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/player.rb'
+class Team
+  attr_reader :country,
+              :players,
+              :player
+              :eliminated
```

Comment:
Think this needs to be an `attr_writer` to allow users to access this value from outside the class.